BridgeSDK/BridgeAPI/SBBAuthManager.m

```diff
@@ -363,6 +363,34 @@ - (NSURLSessionDataTask *)signOutWithCompletion:(SBBNetworkManagerCompletionBloc
     }];
 }
 
+- (NSString *)savedUsername
+{
+    NSString *username = nil;
+    if (_authDelegate) {
+        if ([_authDelegate respondsToSelector:@selector(usernameForAuthManager:)]) {
```

Comment:
by design, yes, and documented; see the doc comments for SBBAuthManagerDelegateProtocol and its optional methods in SBBAuthManager.h for details.